website/src/controllers/groups.js

```diff
@@ -418,7 +436,10 @@ api.join = function(req, res, next) {
 
   if (!_.contains(group.members, user._id)){
     group.members.push(user._id);
-    group.invites.splice(_.indexOf(group.invites, user._id), 1);
+    //Can we have someone joing without an invite? I added this for testing since we join users without an invite
+    if (group.invites.length > 0) {
```

Comment:
The change is fine but I think we could have discovered a serious security issue. I don't want to expose it here publicly so I'll write directly to @crookedneighbor